### ISSUE TITLE
hanami server 2.0.3 on Windows 10 fails with "worker mode not supported" error

### ISSUE THREAD

**Issue by pcopissa** [hanami/hanami#1285]
On Windows 10 64bits follow the steps in https://guides.hanamirb.org/v2.0/introduction/getting-started/
That is:
1) Install Ruby 3.2.1 (using RubyInstaller)
2) In a Ruby-enabled ```cmd``` prompt, install hanami 2.0.3 via ```ridk exec get install hanami```.
3) In a Ruby-enabled ```cmd``` prompt, create a new project via ```hanami new test```. Manually finish the installation with ```bundle install``` (to install missing ```rack-test```)
4) In a Ruby-enabled DOS prompt, start webserver (Puma presumably) via ```bundle exec hanami server```
5) In browser, enter URL ```http://localhost:2300```
6) Observed result: The browser displays an error message ```Connection failed```. Furthermore the DOS prompt in step 2 shows: ```ERROR: worker mode not supported on ruby on this platform```
7) Expected result (according to the "Getting started" page): The browser displays ```Hello from Hanami```

**Comment by carlosm27** [comment]
Could you try to go to the `config/puma.rb` file and comment on the line `workers     ENV.fetch("HANAMI_WEB_CONCURRENCY", 2)`. That works for me.

**Comment by timriley** [comment]
Thanks for adding this detail, @carlosm27. From the sounds of it, if we just use a single worker by default, we should be OK on Windows?

**Comment by carlosm27** [comment]
Yes, I just tried on Windows 11. I think it will work also on Windows 10. I got the solution from this [issue](https://github.com/MartinStyk/quotes-recommender/issues/4).
